build/push-okd4.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+validateOC() {
+  OC_USER_NAME=$(oc whoami 2>/dev/null)
+  if [ ! $? -eq 0 ]; then
+    echo "You must login with 'oc login' or set KUBECONFIG env variable"
+    exit 1
+  fi
+}
+
+validateOC
+
+DOCKER_CERTS_FOLDER="/etc/docker/certs.d/"
+REGISTRY_ROUTE=$(oc get route default-route -n openshift-image-registry -o json | jq -r '.spec.host')
+OC_PROJECT_NAME=$(oc project -q)
+PROJECT_NAME=${PROJECT_NAME-$OC_PROJECT_NAME}
+VERSION=$(git describe --tags --always --dirty)
```

Comment:
Can we make it configurable as in build.sh?
`VERSION=${RELEASE_NAME-$(git describe --tags --always --dirty)}`

Reply:
Thanks, done